src/api/entities/UserToParticipantRole.ts

```diff
@@ -0,0 +1,39 @@
+import { Model } from 'objection';
+
+export class UserToParticipantRole extends Model {
+  static get tableName() {
+    return 'usersToParticipantRoles';
+  }
+  static get idColumn() {
```

Comment:
the PK of this table is the combination of all three columns, which we have to specify here